test/certs.js

```diff
@@ -63,35 +63,17 @@ function getCertificate(domain) {
   });
 }
 
-async function test() {
+describe('certificate expiry date', function() {
   const now = Date.now();
 
-  // start all the requests in parallel
-  const requests = DOMAINS.map(domain => [domain, getCertificate(domain)]);
-
-  let ok = true;
-
-  for (const [domain, request] of requests) {
-    let status = 'OK';
-    try {
-      const cert = await request;
+  for (const domain of DOMAINS) {
+    it(domain, async function() {
```

Comment:
Nit: I like using `specify` here instead of `it`; they are equivalent but since you're not doing `it("should do X", ...)` it reads better.

Reply:
Good, I didn't like `it` or understand what it meant, but didn't know there was an option.